src/System.Net.Http/src/System/Net/Http/Headers/KnownHeaders.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
+using System.Collections.Generic;
```

Comment:
Do you still need these 2 'using' statements now that you aren't doing `HashSet`?

Reply:
@stephentoub is there an analyzer we can enable for this?

Reply:
>is there an analyzer we can enable for this?

Yes, but at least from my experience it caused a fair amount of noise due to ifdefs and projects that built for multiple configurations, where one build would say a using wasn't needed but then another build would require it.  As such I've avoided turning it on.  A few months back I did a pass to run it, remove everything that wasn't needed, and manually put back the ones that then caused compilation failures.  We could do that again, or experiment again with turning on additional rules.